server/rpc_server.go

```diff
@@ -41,6 +55,64 @@ func NewRPCServer(security config.Security) *grpc.Server {
 	if s == nil {
 		s = grpc.NewServer()
 	}
-	diagnosticspb.RegisterDiagnosticsServer(s, &sysutil.DiagnoseServer{})
+	globalDomain = dom
+	// For redirection the cop task.
+	mocktikv.TiDBRPCServerCoprocessorHandler = HandleCopDAGRequest
+	srv := &rpcServer{}
+	diagnosticspb.RegisterDiagnosticsServer(s, srv)
+	tikvpb.RegisterTikvServer(s, srv)
 	return s
 }
+
+// rpcServer is TiDB RPC Server, it reuse the TikvServer interface, but only support the Coprocessor interface now.
+// rpcServer contains below 2 services:
+// 1. Diagnose service, it's used for SQL diagnose.
+// 2. Coprocessor service, it reuse the TikvServer interface, but only support the Coprocessor interface now.
```

Comment:
The `2. ...` contains some duplicated content with the above comment.

Reply:
done.